src/main/webapp/script.js

```diff
@@ -79,14 +79,15 @@ async function displayTab() {
 
         //answer button
         if (rep.trim() == "true" && addTab != "true") {
```

Comment:
=== and !== because checkign for strong equality right? 
any non empty string is truthy so this would work for anything

Reply:
its been functioning properly thus far cause the answer box doesn't show up for user's

Reply:
also i don't think it's true that a nonempty string == "true" or true

Reply:
yee this would work if theyre both strings but strong equality may be better to avoid undefined behavior

Reply:
might as well leave it if youre sure its always gonna be a string